docker/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # ARG required due to build option in docker-compose
 ARG ENV_MODE
 
-FROM ubuntu:18.04
+FROM python:3.7-slim-buster as builder
```

Comment:
Didn't we say that we use 3.8 in production in the end?

Reply:
Yes, commit was not yet pushed, done now.